aad.js

```diff
@@ -0,0 +1,58 @@
+function Murphy() {
+    alert("Murphy, Boxer, $123.45");
+  }
+
+function Poppy() {
+    alert("Poppy, Italian Anklebiter, $123.45");
+  }
+
+function Jack() {
+    alert("Jack, Gargantuan Dwarf Hound, $123.45")
+}
+
+function Duffy() {
+    alert("Duffy, Telepathic Cur, $123.45")
+}
+
+function Lucas() {
+    alert("Lucas, White-Nosed Goof, $123.45")
+}
+
+function Jake() {
+    alert("Jake, Blueblood Fetcher, $123.45")
+}
+
+function Angus() {
+    alert("Angus, Moordwelling Deathprowler, $123.45")
+}
+
+function Violet() {
+    alert("Violet, Lab, $123.45")
+}
+
+function Piper() {
+    alert("Piper, Scotish Shedder, $123.45")
+}
+
+function Maximus() {
```

Comment:
Great, but try to think of a way you can reuse a function and take in data that you've passed in from your click handler...kind of like

https://codepen.io/jimmynono/pen/RmbzQb?editors=1010